Corona-Warn-App/src/test/java/de/rki/coronawarnapp/presencetracing/checkins/checkout/ContactJournalCheckInEntryCreatorTest.kt

```diff
@@ -36,13 +39,13 @@ class ContactJournalCheckInEntryCreatorTest : BaseTest() {
         type = 1,
         description = "Restaurant",
         address = "Around the corner",
-        traceLocationStart = Instant.parse("2021-03-04T22:00+01:00"),
-        traceLocationEnd = Instant.parse("2021-03-04T23:00+01:00"),
+        traceLocationStart = Instant.parse("2021-03-04T22:00:00Z"),
```

Comment:
How come you changed this? this may mask time zone conversion issues

Reply:
it does not parse as Java Instant, I will see what I can do about it